src/App.js

```diff
@@ -1,10 +1,30 @@
-import './App.css';
+import React from "react";
+import {
+    BrowserRouter as Router,
+    Routes,
+    Route,
+} from "react-router-dom";
+import Home from "./pages";
+import Page2 from "./pages/Page2.js";
 
 function App() {
-  return (
-    <div className="App">
-    </div>
-  );
+    return (
+        <Router>
+            <Routes>
+                <Route exact path="/" element={<Home />} />
+                <Route path="/Page2" element={<Page2 />} />
```

Comment:
```suggestion
                <Route path="/Page2" element={<ProfilePage />} />
```
Give this file a meaningful name. As a best practice we don't use these kind of names